pkg/kubectl/cmd/get.go

```diff
@@ -150,6 +150,9 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 	allNamespaces := cmdutil.GetFlagBool(cmd, "all-namespaces")
 	showKind := cmdutil.GetFlagBool(cmd, "show-kind")
 	mapper, typer := f.Object()
```

Comment:
the test MixedFactory was actually returning a typer that would recognize the test types, but the point of `TestGetUnknownSchemaObjectListGeneric` is to test serialization of objects the compiled-in typer *doesn't* recognize. Because you plumbed the typer correctly, we needed a way to simulate the typer not recognizing the test type. 

fixed in https://github.com/mfojtik/kubernetes/pull/1, and removed this field and test-only hack